Updated IDs and added two missing beers

johnjenkman opened the pull request:
Updated all IDs to match the recipe ID number from DIY Dog 2018.

johnjenkman committed:
Updated IDs

Updated all IDs to match the recipe ID number from DIY Dog 2018.

johnjenkman committed:
Added Kingpin and Mashtag 2016

These two beers were missing but were marked as complete in the read me.

johnjenkman committed:
Rename mashtag2016 to mashtag2016.json

johnjenkman committed:
Fixed issue with image url

Null value for image url caused test to fail.

samjbmason commented:
Thanks for doing this, however, updating all the ID's like this might break other peoples app's so I would be happy to add the new recipes but cant switch the ID's

sammdec reviewed (changes requested):
As mentioned in my comment I cant change the ID's until a potential V3

johnjenkman reviewed (commented):
No worries. All good. With the release of DIY Dog 2019 later this year it would be good to create a V3 branch and fix this up.

johnjenkman committed:
Revert "Updated IDs"

This reverts commit 1cb4dd4ff9550ea753c23e7295ed368867d48386.

johnjenkman committed:
Fixed non unique id issue for Kingpin and Mashtag 2016

After reverting the update to all beer IDs these two beers had non
unique IDs. Corrected IDs to fix failing build.
